extern/Makefile.in

```diff
@@ -1,8 +1,9 @@
-# Makefile.in generated by automake 1.14.1 from Makefile.am.
+# Makefile.in generated by automake 1.11.1 from Makefile.am.
```

Comment:
Because I didn't bother to run `autoreconf` in a virtual machine with `ubuntu` 14 or 15. It does not matter too much, but we should probably rerun `autoreconf` from a decently new distro when dust have settled.